src/patternfly/components/Tabs/tabs.scss

```diff
@@ -66,6 +66,10 @@ $pf-v6-c-tabs--spacer-map: build-spacer-map("none", "sm", "md", "lg", "xl", "2xl
   --#{$tabs}__link--FontSize: var(--pf-t--global--font--size--sm);
   --#{$tabs}__link--BackgroundColor: var(--pf-t--global--background--color--action--plain--default);
   --#{$tabs}__link--BorderRadius: var(--pf-t--global--border--radius--small);
+  --#{$tabs}__link--BorderWidth: var(--pf-t--global--high-contrast--border--width--regular);
```

Comment:
could may be use the new --pf-t--global--border--width--action--plain--hover token here instead?